docs/get-started/vue3-vite.md

```diff
@@ -277,14 +277,30 @@ The definition above will generate the following controls:
 
 ### Limitations
 
-`vue-component-meta` cannot currently reference types from an import alias. You will need to replace any aliased imports with relative ones, as in the example below. See [this issue](https://github.com/vuejs/language-tools/issues/3896) for more information.
+If your `tsconfig.json` file uses `references` to other config files like `tsconfig.app.json`, `tsconfig.node.json` etc., you should define the tsconfig file used for your main application code via the framework option shown below so `vue-component-meta` can use it properly.
+Otherwise, you might face missing component types/descriptions or unresolvable import aliases like `@/some/import`.
```

Comment:
@larsrickert, when you have a moment, can you adjust the wording, links, and example to the following to provide some clarity on this known limitation:
````suggestion
If you're working with a project that relies on the [`references`](https://www.typescriptlang.org/docs/handbook/project-references.html) field to link to other existing configuration files (e.g., `tsconfig.app.json`, `tsconfig.node.json`), we recommend that you update your [`.storybook/main.js|ts`](../configure/index.md) configuration file and add the following:

```ts
// .storybook/main.ts
import type { StorybookConfig } from '@storybook/vue3-vite';

const config: StorybookConfig = {
  framework: {
    name: '@storybook/vue3-vite',
    options: {
      docgen: {
        plugin: 'vue-component-meta',
        tsconfig: 'tsconfig.app.json',
      },
    },
  },
};

export default config;
```

<Callout variant="info">

This is not a limitation of Storybook but instead from the `vue-component-meta` package. For more information, refer to the appropriate [GitHub issue](https://github.com/vuejs/language-tools/issues/3896).

</Callout>

When Storybook starts, it will be able to infer the types, descriptions, and other metadata from your Vue components and display them in the Storybook UI using the `vue-component-meta` plugin.
````

Reply:
@jonniebigodes Thanks for the suggestion! I updated the docs, can you please take another look? :)

Reply:
@larsrickert appreciate you addressing the feedback so promptly. I've checked, and just one small item, and this is good to go. Let me know and we'll take it from there.